### PR TITLE
phpcs resolved in controllers template folder files 

### PR DESCRIPTION
**Pull request by vaishaliagola27**

### CODE REVIEW

File: app/main/controllers/template/RTMediaAJAX.php [hunk 1]
```diff
@@ -1,37 +1,49 @@
...
-	function create_album() {
-		$nonce = filter_input( INPUT_POST, 'create_album_nonce', FILTER_SANITIZE_STRING );
-		$_name = filter_input( INPUT_POST, 'name', FILTER_SANITIZE_STRING );
+	/**
+	 * Create album.
+	 */
+	public function create_album() {
+		$nonce        = filter_input( INPUT_POST, 'create_album_nonce', FILTER_SANITIZE_STRING );
+		$_name        = filter_input( INPUT_POST, 'name', FILTER_SANITIZE_STRING );
 		$_description = filter_input( INPUT_POST, 'description', FILTER_SANITIZE_STRING );
```
Comment (chandrapatel) [inline app/main/controllers/template/RTMediaAJAX.php]: @vaishaliagola27 Sanitize it using `sanitize_text_field()` function if see `FILTER_SANITIZE_STRING` in filter_input() function. Do same at other places as well.

`$_description = sanitize_text_field( filter_input( INPUT_POST, 'description', FILTER_SANITIZE_STRING ) );`

File: app/main/controllers/template/rtmedia-actions.php [hunk 2]
```diff
@@ -166,19 +166,18 @@ function rtmedia_gallery_options() {
 		}
 
 		if ( ! empty( $output ) ) {
-			echo $output; // @codingStandardsIgnoreLine
+			echo $output;
```
Comment (chandrapatel) [inline app/main/controllers/template/rtmedia-actions.php]: Don't we need to add `// phpcs:ignore <rule-name>`?

Comment (chandrapatel) [reply app/main/controllers/template/rtmedia-actions.php]: Check at other places as well.

File: app/main/controllers/template/RTMediaAJAX.php (hunk 1 above)
Comment (vaishaliagola27) [reply app/main/controllers/template/RTMediaAJAX.php]: I'll Address this feedback in new PR for the same files.

File: app/main/controllers/template/rtmedia-actions.php (hunk 2 above)
Comment (vaishaliagola27) [reply app/main/controllers/template/rtmedia-actions.php]: I'll Address this feedback in new PR for the same files.

File: app/main/controllers/template/RTMediaAJAX.php (hunk 1 above)
Comment (vaishaliagola27) [reply app/main/controllers/template/RTMediaAJAX.php]: @chandrapatel Please check. I've addressed this feedback in same PR.

**Comment by chandrapatel** [comment]
@vaishaliagola27 I have approved PR

**Pull request merged**